promptsource/templates/app_reviews/templates.yaml

```diff
@@ -58,39 +46,24 @@ templates:
     name: generate_review
     reference: Generate a review from the rating.
   9746ce4b-ac58-4dfb-9783-d77c95cb62cf: !Template
-    answer_choices: null
-    answer_choices_key: null
+    answer_choices:
+    - "\u2605"
+    - "\u2605\u2605"
+    - "\u2605\u2605\u2605"
+    - "\u2605\u2605\u2605\u2605"
+    - "\u2605\u2605\u2605\u2605\u2605"
+    answer_choices_key: star
     id: 9746ce4b-ac58-4dfb-9783-d77c95cb62cf
-    jinja: 'What would be the *-rating of this review (* being lowest and ***** being
-      highest) : {{review}}?
-
-      |||{% if star==5 %}
-
-      *****
-
-      {% elif star==4%}
-
-      ****
-
-      {% elif star==3%}
-
-      ***
-
-      {% elif star==2%}
-
-      **
-
-      {% else %}
-
-      *
-
-      {% endif %}'
+    jinja: "What would be the \u2605-rating of this review (\u2605 being the lowest\
+      \ and \u2605\u2605\u2605\u2605\u2605 being the highest) : {{review}}?\n|||{{answer_choices[star-1]}}"
```

Comment:
I think this kind of expression (or something like "on a scale of 1 to 5") doesn't count as `choices_in_prompt`, but I can be totally mistaken.